frontend/src/components/chat/question-suggestion-cards.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { Card } from "@/components/ui/card";
+import { SuggestedQuestions } from "@/lib/types";
+
+type QuestionSuggestionCardsProps = {
+  suggestions?: SuggestedQuestions;
+  onSubmit: (suggestionInput?: string) => void;
+};
+
+export default function QuestionSuggestionCards({
+  suggestions,
+  onSubmit,
+}: QuestionSuggestionCardsProps) {
+  return (
+    <div className="grid grid-cols-1 gap-4">
```

Comment:
IMO if you want to have just 1 column you might as well just use`flex-col` flexbox:)